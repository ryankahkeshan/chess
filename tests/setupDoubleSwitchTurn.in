setup
+ K e1
+ k e8
= blaCk
done
setup
+ P e2
+ p e7
= White
done
game human human
move e7 e6
move e2 e4
move e7 e5
resign
