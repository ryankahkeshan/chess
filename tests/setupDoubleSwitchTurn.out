8  _ _ _ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ p _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ p _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2  _ _P_ _
1 _ _ K _ 

  abcdefgh

Invalid move
8  _ _k_ _
7 _ _ p _ 
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

8  _ _k_ _
7 _ _ _ _ 
6  _ _ _ _
5 _ _ p _ 
4  _ _P_ _
3 _ _ _ _ 
2  _ _ _ _
1 _ _ K _ 

  abcdefgh

Black wins!
Final Score:
White: 0
Black: 1
